cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/README.md

```diff
@@ -0,0 +1,32 @@
+# Package Name
+oradb_discovery_v3_6_11_19.sql
+
+## Description
+The Database discovery & analysis tool is a SQL utility that captures necessary details of an oracle database in order to analyse on-Prem or cloud Oracle Databases for the suitability of migration/upgrades. This utility runs on Non Container or Container databases and helps in generating HTML report with detailed description
+
+## Usage
+sqlplus "/ as sysdba"  @oradb_discovery_v3_6_11_19.sql
+
+### Step 1
+This is a discovery utility for Oracle database compatible for 11g onwards with a below features:
+1.	Current version of the script is v3.6 and it supports Oracle 11g or above database
+2.	Single version script for both Oracle Multitenant and  Non-Multitenant database
+3.	This is a platform independent script as we need sqlplus "/ as sysdba"  to execute only
+
+This script needs to execute on Oracle database server with a user who has “/as sysdba” access
+•	Copy the oradb_discovery_v3_6_11_19.sql file to the server
+•	To execute the script 	:  sqlplus "/ as sysdba"  @oradb_discovery_v3_6_11_19.sql
```

Comment:
Should read "To execute the script:"
Should read `sqlplus "/ as sysdba"  @oradb_discovery_v3_6_11_19.sql`

Reply:
Submitted with required changes.